SlowCheetah.JDT/Processors/Attributes/AttributeExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace SlowCheetah.JDT
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Linq;
+
+    /// <summary>
+    /// Implements extensions for <see cref="JdtAttributes"/>
+    /// </summary>
+    internal static class AttributeExtensions
```

Comment:
Can you rename this to `JdtAttributeExtensions` - just so it is not confused with general C# attributes.